test/fixtures/action_text/rich_texts.yml

```
@@ -0,0 +1,9 @@
+rich_text_first_pdf_template:
+  record: first_pdf_template (PdfTemplate)
+  name: 'content'
+  body: "<div class=\"trix-content\">\n  <div>A example of pdf template</div><pre>2+2</pre><div>a variable</div><pre>var</pre><div>final</div>\n</div>\n"
```

```suggestion
  body: |
    <div class="trix-content">
      <div>A example of pdf template</div>
      <pre>2+2</pre>
      <div>a variable</div>
      <pre>var</pre>
      <div>final</div>
    </div>
```
Quizás acá convenga poner esta notación (la forma quizás la quieras modificar, digo para que se entienda un poco mejor que el string de corrido).

---

olvidate, me parece perfecto, no sabia de esta forma para los fixtures.